CONTRIBUTING.md

```diff
@@ -1,174 +1,163 @@
-	want to split discussion about an issue across multiple locations.
-	- If an issue was closed due to it being fixed, you may still open a new
-	issue describing the new regression.
-- Only if you can't find an existing issue, you may open a new issue describing
-your bug.
-	- Your title should be clear, concise, and adequately summarise the issue.
-	Titles like 'Why is this broken?' and 'my code doesn't work' aren't helpful.
-	- Include as much relevant information as possible. We can't diagnose the
-	issue if we don't know anything about it.
-	- Please include simple, clear and easy-to-follow reproduction steps. You
-	may include a .rbxl file if you need some existing non-trivial setup.
-- When your issue is noticed by a project maintainer, it will be labelled
-appropriately.
-	- Bugs start out labelled as **'status: evaluating'**. This occurs as soon
-	as we notice the issue. At this stage, we have not verified the bug for
-	ourselves.
-	- When the bug is verified, it will be labelled as **'status: needs design'**
-	if there is not an obvious or immediately actionable solution. This often
-	occurs with bugs whose fixes would be breaking changes.
-	- Once a solution has been identified and approved. it will be labelled as
-	**'status: approved'**. Approved bugs will be fixed in a future version of
-	Fusion, and will be closed once the fix ships.
-- Be patient - all project maintainers are voluteers, so it will likely take
-some time to respond.
-	- Don't bump or chase up your issue - commenting 'Any updates on this?' does
-	not add value to the conversation, and only clutters the issue.
+# Contributions
+
+We love contributions and suggestions from you guys! To make sure everything goes as smoothly as possible, we've set up a few processes you can follow for different contribution types:
+
+-----
+
+## I need help or have a question
+
+Whatever you're looking for, here are some resources for you to get help!
+
+- [The Roblox OSS Discord](https://discord.gg/h2NV8PqhAD) has a [#fusion](https://discord.com/channels/385151591524597761/895437663040077834) channel
+- Check out [our Discussions page](https://github.com/Elttob/Fusion/discussions) on GitHub
+
+Please don't submit issues to the repository to ask a question. Prefer to start a
+discussion instead :)
+
+-----
+
+## I found a bug
+
+### Are you the first person to report it?
+
+- Make sure to search for open issues *and* closed issues - it's possible
+someone else opened an issue about it and then that issue was subsequently
+closed.
+
+- If an issue is found to be a duplicate, it is closed right away. We don't
+want to split discussion about an issue across multiple locations.
+
+- If an issue was closed after a bug fix, but the bug has come back, feel
+free to open a new issue describing the new variant of the bug.
+
+
+### Open a new issue, describing the bug
+
+- Your title should be clear, concise, and adequately summarise the issue.
```

Comment:
This sentence reads awkwardly due to using two adjectives but then switching to a verb.